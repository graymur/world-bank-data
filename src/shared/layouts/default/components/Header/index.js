import React from 'react';
import MainNav from '../MainNav';

const Header = () => (
	<header className='header'>
		<div className='container'>
			<MainNav/>
		</div>
	</header>
);

export default Header;
