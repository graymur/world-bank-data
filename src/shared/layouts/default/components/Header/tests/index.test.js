/* eslint-disable */
import React from 'react';
import renderer from 'react-test-renderer';
import {StaticRouter} from 'react-router';
import Header from '../index';

test('Renders content in wrapper', () => {
	const component = renderer.create(
		<StaticRouter location="someLocation" context={{}}>
			<Header/>
		</StaticRouter>
	);

	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});
