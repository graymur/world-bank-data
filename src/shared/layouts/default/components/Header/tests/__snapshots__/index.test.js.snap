// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders content in wrapper 1`] = `
<header
  className="header"
>
  <div
    className="container"
  >
    <nav
      className="main-nav"
    >
      <a
        aria-current={false}
        className="main-nav__item"
        href="/"
        onClick={[Function]}
        style={undefined}
      >
        Main page
      </a>
      <a
        aria-current={false}
        className="main-nav__item"
        href="/countries"
        onClick={[Function]}
        style={undefined}
      >
        Countries
      </a>
      <a
        aria-current={false}
        className="main-nav__item"
        href="/indicators"
        onClick={[Function]}
        style={undefined}
      >
        Indicators
      </a>
    </nav>
  </div>
</header>
`;
